Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: build-sdk
+build-sdk:
```

Comment:
I'd like to keep these out of the repository and potentially move this to the kratos main directory or something. Otherwise we have multiple places where we need to update this code if anything ever changes. Can you please revert these SDK pipeilne changes please?